github-workflow-roles/main.tf

```diff
@@ -4,8 +4,9 @@
 ################################################################################
 locals {
   aws_account_id = var.aws_account_id != "" ? var.aws_account_id : data.aws_caller_identity.current.account_id
+  rvm_account_id = var.rvm_account_id != "" ? var.rvm_account_id : null
 
-  github_organization_name = var.principal_type == "github" ? var.github_organization_name : null
+  github_organization_name = var.principal_type == "github" || var.principal_type == "breakglass" ? var.github_organization_name : null
```

Comment:
fun fact, in the latest version of TF, your condition validation blocks can reference other variable values. So you could specify that `github_organization_name` must not be null if `principal_type` is set to `breakglass` or `github`.